### PR TITLE
[#173035929] defined spec first draft

### PR DESCRIPTION
**Pull request by balanza**
* Defined `SecurityDefinition`
* fixed `fiscalcode` param name
* fixed some definitions
* changed express handler path accordingly 

@BurnedMarshal please check definitions are correct!

### CODE REVIEW

**Comment by gunzip** [comment]
lgtm. @BurnedMarshal ?

File: EligibilityCheck/index.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ const app = express();
 secureExpressApp(app);
 
 // Add express route
-app.get("/v1/bonus/eligibility/:fiscalcode", EligibilityCheck());
+app.get("/api/v1/bonus/eligibility/:fiscalcode", EligibilityCheck());
```
Comment (BurnedMarshal) [inline EligibilityCheck/index.ts:13]:
```suggestion
app.post("/api/v1/bonus/eligibility/:fiscalcode", EligibilityCheck());
```

File: openapi/index.yaml [hunk 2]
```diff
@@ -66,3 +79,149 @@ definitions:
...
+        type: integer
+        minimum: 0
+        maximum: 50000
+    required:
+      - id
+      - status
+      - max_amount
+      - max_tax_benefit
+      - family_members
+  QrCode:
```
Comment (BurnedMarshal) [inline openapi/index.yaml:141]: QrCode is generated backend side

Comment (balanza) [reply openapi/index.yaml:141]: Ok removed. Actually I left some definitions not used yet, but I guess they'll be needed in future operations. Please review the definitions when you'll add a new operation

**Pull request merged**